packages/base/src/tools.ts

```diff
@@ -867,3 +867,24 @@ export const stringToArrayBuffer = async (
   );
   return await base64Response.arrayBuffer();
 };
+
+export const getNumericFeatures = (
+  featureProperties: Record<string, Set<any>>
+) => {
```

Comment:
```suggestion
export const getNumericFeatureAttributes = (
  featureAttributes: Record<string, Set<any>>
) => {
```

It's common to refer to the structured data attached to a feature as "attributes". That might help with name collisions with React terminology.